src/main/java/no/sikt/sws/SnapshotTakingLambda.java

```diff
@@ -0,0 +1,52 @@
+package no.sikt.sws;
+
+import com.amazonaws.http.HttpMethodName;
+import com.amazonaws.services.lambda.runtime.Context;
+import no.sikt.sws.exception.SearchException;
+import nva.commons.apigateway.ApiGatewayHandler;
+import nva.commons.apigateway.RequestInfo;
+import nva.commons.apigateway.exceptions.ApiGatewayException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.sql.Timestamp;
+
+import static no.sikt.sws.constants.ApplicationConstants.BACKUP_BUCKET_NAME;
+
+public class SnapshotTakingLambda extends ApiGatewayHandler<Void, String> {
+
+    private static final Logger logger = LoggerFactory.getLogger(RegisterSnapshotRepoHandler.class);
```

Comment:
Same issue as you faced earlier - you are using a logger for another class (RegisterSnapshotRepoHandler)

Reply:
Considered this one